lua/tardis/libraries/libraries/sh_settings.lua

```diff
@@ -121,11 +121,25 @@ function TARDIS:GetSetting(id, src)
         ply = (src:IsPlayer() and src) or src:GetCreator()
     end
 
+    local function list_value_exists(setting_data, value)
```

Comment:
I'm not entirely sure what's causing this but I got this a couple times after loading your branch when I spawned a TARDIS, although seemed to go away after a bit so couldn't debug any further:

Clientside error by the way
```
[tardis] addons/tardis/lua/tardis/libraries/libraries/sh_settings.lua:405: attempt to call method 'CallCommonHook' (a nil value)
  1. OnSettingChanged - addons/tardis/lua/tardis/libraries/libraries/sh_settings.lua:405
   2. func - addons/tardis/lua/tardis/libraries/libraries/sh_settings.lua:364
    3. unknown - lua/includes/extensions/net.lua:33
```

Not sure if you know what might have caused this but may be worth adding a check to ensure the function exists before calling it, looks like it's being called before it's finished initialising

Reply:
added